tools/packaging/packaging.psm1

```diff
@@ -2245,7 +2247,9 @@ function Test-FileWxs
         [string] $HeatFilesWxsPath
     )
 
-    [xml] $filesAssetXml = Get-Content -Raw -Path $FilesWxsPath
+    $filesAssetString = (Get-Content -Raw -Path $FilesWxsPath).Replace('$(var.FileArchitecture)',$env:FileArchitecture)
```

Comment:
It would be helpful adding a comment to this line of script explaining why replacing `$(var.FileArchitecture)` here.